File: Ejercicios PHP/5.Condicionales If Else.php
<?php
$a=2;
if ($a ==2){
    echo("a vale 2. \n");
}else{
    echo('a es diferente de 2');
}
//Concatenar condiciones
$Nombre = 'Esteban';
$Edad = 28;
$Pais='Colombia';

if ($Nombre =='Esteban' && $Edad == 28 && $Pais =='Colombia'){
    echo("Su nombre es $Nombre, tiene $Edad años, y es de $Pais"); 
}
elseif($Nombre == 'Esteban' && $Edad!==28){
    echo('Su nombre es Esteban y no tiene 28 años');
}elseif($Nombre != 'Esteban' && $Edad == 28){
    print('Su nombre no es Esteban y tiene 28 años');
}else{
    echo('No se llama Esteban ni tiene 28 años');  
}
//Para ejecutar en terminal: php "C:\Sena\Analisis y desarrollo de software\Ficha 3066478 segundo trimestre\Ejercicios PHP\5.Condicionales If Else.php" 
?>

File: Ejercicios C/5.Condicionales If Else.cpp
#include<iostream>
#include<string>

using namespace std;

int main(){
    int a;
    a=2;
    if (a=2){
        cout<<"A vale 2"<<endl;
    }
    else{
        cout<<"A es diferente a 2"<<endl;
    }
string nombre, pais;

nombre="Esteban";
int edad=27;
pais="Colombia";

if (nombre=="Esteban" && edad==27 && pais=="Colombia"){
    cout<<"Su nombre es "<< nombre <<" tiene "<< edad <<" y es de "<< pais <<endl;
}else if(nombre=="Esteban" && !(edad==27)){
    cout<<"Su nombre es Esteban y no tiene 27 años"<<endl;;
}else if(!(nombre=="Esteban") && (edad==27)){
    cout<<"Su nombre no es Esteban y tiene 27 años"<<endl;
}else{
    cout<<"No se llama Esteban ni tiene 27 años"<<endl;
}
}